src/Resources/ui/scss/structures/_cookie-bar.scss

```diff
@@ -4,18 +4,19 @@
 
 .kmcc-cookie-bar {
     position: fixed;
-    bottom: 0;
-    left: 0;
-    right: 0;
 
-    height: 100%;
+    bottom: 8px;
+    left: 8px;
+    width: calc(100vw - 16px);
+
+    height: auto;
     overflow: auto;
 
     display: none;
 
     z-index: $kmcc_z-cookie-bar;
 
-    padding: 35px 0 35px;
+    padding: 24px;
 
```

Comment:
@sector32 Can you reorder the properties a bit? So all box-modal props are together etc..
(use https://jsbin.com/dufujufide/edit?css as a guideline)

@diskwriter Should we add our frontend-linters to this project? Or overkill?

Reply:
@dbeerten No we should definitely add them.
@sector32 Could you add the Kunstmaan frontend linters to this project? (see https://github.com/Kunstmaan/frontend-linters)